game/family.cc

```diff
@@ -0,0 +1,16 @@
+#include <vector>
+
+#include "type.h"
+#include "family.h"
+
+using namespace std;
+
+Family::Family(const Type type): type{type} {}
+
+size_t Family::effectiveness(vector<Family> families) {
+  int strength = 1;
+  for(auto otherFamily : families) {
```

Comment:
excuse me sir there needs to be a space after the `for` in the loop pls and thank you do you even code 👎

Reply:
This is unacceptable
![bitmoji](https://render.bitstrips.com/v2/cpanel/4d4d136c-c127-42d8-8684-1ecff186cbca-29b03f31-567b-40a9-836c-bbce3d642229-v1.png?transparent=1&palette=1&width=246)